docs/release/handbook.md

```diff
@@ -0,0 +1,292 @@
+# Kubeflow Release Handbook
+
+## Executive Summary
+
+The purpose of this document is to define the roles and processes for the Kubeflow community to release Kubeflow. This document will attempt to coordinate releasing between the five major components and working groups in Kubeflow, as well as defining best practices, roles, and processes to make releasing Kubeflow easier for the community with each iteration.
+
+## People and Roles
+
+### Release Manager
+The Release Manager will be responsible for coordinating the release and taking ultimate accountability for all release tasks to be completed on time.
+
+**Prerequisites:**
+1. Be a contributor for at least 2 years
+2. Have shadowed in a previous release or be a Release Team member in the past
+3. Contributions in at least 2 Working Groups
+
+**Time Commitments:** This role will need to be synchronous. The Release Manager is expected to be joining all the weekly and burndown meetings as well as coordinate with the Release Team and the Working Group leads. For the Development phase an estimate of the work needed is 4-8 hours per week. For the next phases the Release Manager will need to devote more time and coordinate the tasks of the release.
+
+**Responsibilities:**
+* Providing updates to the mailing list with the progress of the release
+* Coordinating directly with the WG liaisons and leads about dates and deliverables
+* Coordinating with the Release Team Members for the progress of the release
+* Ensuring the WG leads have cut the necessary github branches and tags for the different phases of the release
+* Driving the Release Team meetings
+* All the responsibilities of a Release Team Member
+
+### Release Team Member
+Release Team Members ensure that there's enough bandwidth to perform release related tasks. They work closely with the Release Manager to address release issues.
+
+**Prerequisites:** Have contributions in at least 2 Working Groups
```

Comment:
I am not sure this should be a prerequisite. In the same light as @thesuperzapper's [above comment ](https://github.com/kubeflow/manifests/pull/1907/files#r658416150)

Reply:
Yes, I agree.

I pushed a commit that completely removes this requirement for members, as also discussed in https://github.com/kubeflow/manifests/pull/1907#discussion_r658416150 and https://github.com/kubeflow/manifests/pull/1907#discussion_r653852378